tests/test_maya_interval.py

```diff
@@ -570,3 +570,16 @@ def test_interval_from_iso8601_duration():
 
     assert interval.start == s
     assert interval.end == e
+
+
+def test_issue_168_regression():
```

Comment:
I'm all ears on how to improve this test. It looks like there aren't any explicit tests for the `maya.intervals(...)` functionality. I'm willing to try adding some if a maintainer will let me know what they'd like to see.

Reply:
IMHO: this test looks good! At least for intervals in a resolution of days. Maybe it makes sense to add tests for other intervals, like seconds, minutes and hours .. If you'd like some of those - I'm always happy to review!